### PR TITLE
Upgrade springfox to 2.6.0

### PR DESCRIPTION
**Pull request by cbornet**
Mainly to get the fix for https://github.com/springfox/springfox/issues/1337

### CODE REVIEW

File: generators/server/templates/_pom.xml [hunk 1]
```diff
@@ -99,7 +99,7 @@
         <%_ if (authenticationType === 'oauth2' || authenticationType === 'uaa') { _%>
         <spring-security-oauth2.version>2.0.9.RELEASE</spring-security-oauth2.version>
         <%_ } _%>
-        <springfox.version>2.5.0</springfox.version>
+        <springfox.version>2.6.0</springfox.version>
```
Comment (jdubois) [inline generators/server/templates/_pom.xml]: You also need to update Gradle

Comment (cbornet) [inline generators/server/templates/_pom.xml]: Indeed !

Comment (cbornet) [inline generators/server/templates/_pom.xml]: Done

**Comment by jdubois** [comment]
I'm merging this before the CI completes, as I'm working on SpringFox right now!

**Pull request merged**